### DISCUSSION TITLE
Periodic Boundary condition for 3D mesh

### DISCUSSION THREAD

**Discussion by haqfariha** [idaholab/moose#24438] [Q&A Getting Started] [answered]
For 2D mesh, I would use 'auto direction = x' or 'y'. But or 3D mesh, for repeating, I need to add periodic BC along plane. Could anyone provide me the format of doing that?
Thanks in advance.

**Comment by GiudGiud** [comment, marked as answer]
Hello

In 3D, it s the same syntax. The auto_direction can be x, y or z. 
No other automatic detection of periodicity in a given direction is supported.

Guillaume
